### PR TITLE
Support .NET custom command

### PR DESCRIPTION
**Pull request by RobiNino**
- [x] All [tests](https://github.com/jfrog/jfrog-azure-devops-extension#testing) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----
Following #468.
image

### CODE REVIEW

File: tasks/JFrogDotnet/task.json [hunk 1]
```diff
@@ -39,17 +39,27 @@
...
+                "custom": "custom"
             }
         },
+        {
+            "name": "customCommand",
+            "type": "string",
+            "label": "Custom command",
+            "defaultValue": "",
+            "helpMarkDown": "The custom command which will be passed to .NET Core CLI for execution. Note that commands that require deploying are not supported, and that build-info collection is not available for custom commands.",
+            "required": true,
```
Comment (yahavi) [inline tasks/JFrogDotnet/task.json:51]: Shoul'd it really be required?

File: tasks/JFrogDotnet/task.json [hunk 2]
```diff
@@ -39,17 +39,27 @@
...
-                "push": "NuGet push"
+                "push": "NuGet push",
+                "custom": "custom"
             }
         },
+        {
+            "name": "customCommand",
+            "type": "string",
+            "label": "Custom command",
+            "defaultValue": "",
```
Comment (yahavi) [inline tasks/JFrogDotnet/task.json]:
```suggestion
```

File: tasks/JFrogDotnet/task.json [hunk 3]
```diff
@@ -39,17 +39,27 @@
...
+        },
         {
             "name": "arguments",
             "type": "string",
             "label": "Arguments",
             "defaultValue": "",
             "helpMarkDown": "The arguments which will be passed to .NET Core CLI for execution. Arguments should be quoted, and space separated.",
             "required": false,
-            "visibleRule": "command = restore"
+            "visibleRule": "command=restore || command=custom"
```
Comment (yahavi) [inline tasks/JFrogDotnet/task.json:61]: If you agree, let's do this also is the other fields below
```suggestion
            "visibleRule": "command = restore || command = custom"
```

**Review by yahavi — CHANGES_REQUESTED** [review]

**Comment by yahavi** [comment]
It might also be challenging to grasp the difference between "custom command" and "arguments" without providing examples of each.

File: tasks/JFrogDotnet/task.json (hunk 1 above)
Comment (RobiNino) [reply tasks/JFrogDotnet/task.json:51]: If the user chose to provide a custom command, then the custom command is required. Don't you agree?

Comment (yahavi) [reply tasks/JFrogDotnet/task.json:51]: Agree. Right.

File: tasks/JFrogDotnet/task.json (hunk 3 above)
Comment (RobiNino) [reply tasks/JFrogDotnet/task.json:61]: I changed them on purpose, I find it way more readable. Why don't you like this change?

**Pull request merged**